DDT_DATA/testData/icu75/message_fmt2_test.json

```diff
@@ -0,0 +1,41 @@
+{
+  "test_type": "message_fmt2",
+  "tests": [
+    {
+      "label": "Literal text: Includes basic literals",
```

Comment:
`label` is the unique id that gets generated by the test gen framework. Pretty much an auto-increment integer starting at `00000`.

If you run the end-to-end script (`generateDataAndRun.sh`) and look into a file like `TEMP_DATA/testData/icu73/collation_test.json`, you can see that. You can also see a field called `test_description` that better fits the purpose intended here.